addons/mail/static/src/components/message/message.js

```diff
@@ -45,6 +45,10 @@ export class Message extends Component {
          * replace prettyBody with new value in JS (which is faster than t-raw).
          */
         this._prettyBodyRef = useRef('prettyBody');
+        /**
+         * Used as Shadow Dom Host to render email templates inside messages.
+         */
+        this._shadowMessageBodyRef = useRef('shadowMessageBody');
```

Comment:
just `shadowBody` (`message` is deduced by component name `Message`)

Reply:
Done!